Redesign website with accessibility in mind

arianvp opened the pull request:
Header moved into main; use a Hero element instead of title above navbar
This should make the content more accessible for screen-readers. 

Website now takes more liberties with the original design, but it's better for accessibility.

Added screenshots of the new look. Don't know how to render a preview of the HTML with github pages in a PR, so let me find something for that so that somebody can check if the markup is now better for screen readers.

![image](https://user-images.githubusercontent.com/628387/85159641-122f9680-b25e-11ea-9e62-cdb2e8aec839.png)
![image](https://user-images.githubusercontent.com/628387/85159815-40ad7180-b25e-11ea-8c6f-ef89e492a9d9.png)

![image](https://user-images.githubusercontent.com/628387/85159759-34291900-b25e-11ea-81d5-35f367d02b50.png)

bramd commented:
@arianvp Cool, thanks for the pR.

I just built the site locally and looks fine to me. I spotted a few accessibility improvements we could do:

1. We could mark the current page in the navigation with `aria-current="page"`, the semantic way of saying this is the current page
1. The logo in the header is now placed using CSS background, which has the disadvantage of disappearing when someone uses a user stylesheet to replace our styles, which doesn't seem that common these days. It also prevents screen reader users from knowing this is a logo. We could fix both by imcluding the logo in a normal image and giving it an alt attribute that includes that this is a logo, or we could leave the background image, add a `role="img"` and a hidden text that says that this is the logo. That would at least fix the screenreader issue. Ensure to put the role on the hidden span that is already there, putting it on the `a` would break the default role of link.

It seems I don't have write access to this repo, otherwise I would just push this changes to your branch. Again, these are fairly minor issues and I think it is fine to merge this even without the suggested changes.

Feel free to ping me if you want any accessibility related feedback to work on the website or app.

arianvp commented:
I added the two suggestions.

Because the logo is not by me but by the https://nl-design-system.gitlab.io/nl-design-system/ component-library,  I'm not going to change it to an `<img>` tag. I added the `role="img"`  attribute to the hidden span instead.    This might be something we want to send as a PR to nl-design-system as well.
